examples/rethinkdb/image/run.sh

```diff
@@ -16,14 +16,21 @@
 
 set -o pipefail
 
+echo Checking for other nodes
 IP=""
-if [[ -n "${KUBERNETES_RO_SERVICE_HOST}" ]]; then
+if [[ -n "${KUBERNETES_SERVICE_HOST}" ]]; then
 
-  : ${NAMESPACE:=rethinkdb}
-  # try to pick up first different ip from endpoints
+  POD_NAMESPACE=${POD_NAMESPACE:-default}
   MYHOST=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
-  URL="${KUBERNETES_RO_SERVICE_HOST}/api/v1/namespaces/${NAMESPACE}/endpoints/rethinkdb-driver"
-  IP=$(curl -s ${URL} | jq -s -r --arg h "${MYHOST}" '.[0].subsets | .[].addresses | [ .[].IP ] | map(select(. != $h)) | .[0]') || exit 1
+  echo My host: ${MYHOST}
+
+  URL="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/namespaces/${POD_NAMESPACE}/endpoints/rethinkdb-driver"
+  echo "Endpont url: ${URL}"
+  echo "Looking for IPs..."
+  token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)  
+  # try to pick up first different ip from endpoints
+  IP=$(curl -s ${URL} --insecure --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt --header "Authorization: Bearer ${token}" \
```

Comment:
Client Version: version.Info{Major:"0", Minor:"20+", GitVersion:"v0.20.2-453-gf2adea25276de8-dirty", GitCommit:"f2adea25276de84232c2a9ea9ef8821a3d3753e9", GitTreeState:"dirty"}
Server Version: version.Info{Major:"0", Minor:"20+", GitVersion:"v0.20.2-453-gf2adea25276de8-dirty", GitCommit:"f2adea25276de84232c2a9ea9ef8821a3d3753e9", GitTreeState:"dirty"}

Checking for other nodes
My host: 10.245.1.7
Endpont url: https://10.0.0.1:443/api/v1/namespaces/default/endpoints/rethinkdb-driver
Looking for IPs...
jq: error: Cannot iterate over null